packages/grafana-toolkit/package.json

```diff
@@ -58,10 +57,6 @@
     "copy-webpack-plugin": "5.1.2",
     "css-loader": "3.4.2",
     "eslint": "7.21.0",
-    "eslint-config-prettier": "7.2.0",
```

Comment:
Do we know what sort of impact (if any) deleting these packages will have on plugin devs?

Reply:
There shouldn't be any impact, as we added `@rushstack/eslint-patch` which lets eslint configs (like `@grafana/eslint-config`) use their own dependencies, which includes `eslint-config-prettier`.